src/factories/request.js

```diff
@@ -62,11 +62,14 @@ const fetchRetry = (
   new Promise((resolve, reject) => {
     const ver = version || config.version || ''
```

Comment:
Now that we check that `ver` is truthy do we need the ` || ''` here?

Reply:
Amended in new commit